### DISCUSSION TITLE
AList各种存储之间自动定时同步，例如自动备份相册到各种网盘

### DISCUSSION THREAD

**Discussion by dr34m-cn** [AlistGo/alist#6807] [Show and tell]
[TaoSync-GitHub](https://github.com/dr34-m/taosync)

点个Star再走吧

![新建](https://raw.githubusercontent.com/dr34-m/taosync/main/README/%E6%96%B0%E5%BB%BA%E4%BD%9C%E4%B8%9A.png)

![任务列表](https://raw.githubusercontent.com/dr34-m/taosync/main/README/%E4%BB%BB%E5%8A%A1%E5%88%97%E8%A1%A8.png)

![任务详情](https://raw.githubusercontent.com/dr34-m/taosync/main/README/%E4%BB%BB%E5%8A%A1%E8%AF%A6%E6%83%85.png)

**Comment by LINJIAXIN-China** [comment]
文件多的话有个弊端，用这个非常占用性能
